### PR TITLE
[5.2.x] Update Azure.Identity to 1.11.4

### LINKED ISSUE #2568: Release a new version with an update ref to 'Azure.Identity'
'Azure.Identity' 1.11.3 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-m5vv-6r4h-3vj9

can you release a version that updates this reference

### PR DESCRIPTION
**Pull request by ErikEJ**
fixes #2568

### CODE REVIEW

File: tools/props/Versions.props [hunk 1]
```diff
@@ -27,8 +27,7 @@
   </PropertyGroup>
   <!-- NetFx and NetCore project dependencies -->
   <PropertyGroup>
-    <AzureIdentityVersion>1.11.3</AzureIdentityVersion>
-    <MicrosoftIdentityClientVersion>4.60.3</MicrosoftIdentityClientVersion>
+    <AzureIdentityVersion>1.11.4</AzureIdentityVersion>
```
Comment (thompson-tomo) [inline tools/props/Versions.props:30]: Thoughts about bumping this 1.12.0

Comment (ErikEJ) [reply tools/props/Versions.props:30]: Actually no, not in a patch

Comment (thompson-tomo) [reply tools/props/Versions.props:30]: @ErikEJ what I do notice is that this is a new dependency from this batch version going forward k

Comment (ErikEJ) [reply tools/props/Versions.props:30]: ?? Sure, but not a new major version... (Or I do not understand what you mean)

Comment (thompson-tomo) [reply tools/props/Versions.props:30]: As it currently stands there is a dependency on Microsoft.Identity however with this pr we are removing that dependency & introducing a new dependency on Azure.Identity hence feel that we should be using the latest.

**Pull request closed without merging**